ModularTegustation/altjobtitles/altjobtitles.dm

```diff
@@ -14,10 +20,10 @@
 	alt_titles = list("Firefighter", "Life Support Specialist")
 	senior_title = "Senior Atmospheric Technician"
 
- //Service
+ // Service
 /datum/job/assistant
 	alt_titles = list("Visitor", "Businessman", "Entertainer")
-	senior_title = "Greytider"
+	ultra_senior_title = "Assistinator"
```

Comment:
oh my god egor

Reply:
Yes. People(including one admin) were in favor of this.

Reply:
<img width="473" alt="image" src="https://user-images.githubusercontent.com/69032410/111730531-bca81900-88c5-11eb-9c7d-cadbeb449000.png">
i love this community